website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-google-gsuite.md

```diff
@@ -3,44 +3,134 @@ title: "Setting up SSO with Google GSuite"
+with a **Client ID** and **Client Secret**. Write these down as you will need them later!
+
+### Enabling the Admin SDK
+
+dbt Cloud requires that the Admin SDK is enabled in this application to request
+Group Membership information from the GSuite API. To enable the Admin SDK for
+this project, navigate to the [Admin SDK Settings page](https://console.developers.google.com/apis/api/admin.googleapis.com/overview)
+and ensure that the API is enabled.
 
 <Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/7f36f50-Screen_Shot_2019-12-03_at_10.15.01_AM.png" title="The 'Admin SDK' page"/>
 
-To finish your setup, you will need to provide support the following values:
-
-- The **Client ID** you generated on the "Create Oauth client ID" page.
-- The **Client Secret** you generated on the "Create Oauth client ID" page.
-- The domain of your Google GSuite domain.
-
-Send these values to your account manager, or via support (either in-app via Intercom, or via email at support@getdbt.com), and we'll get back to you when the integration is ready to use.
+## Configuration in dbt Cloud
+
+To complete setup, follow the steps below in the dbt Cloud application.
+
+### Enable GSuite Native Auth (beta)
+
+- For users accessing dbt Cloud at cloud.getdbt.com, contact your account manager to
+  gain access to the GSuite Native auth configuration UI
+- For users accessing dbt Cloud deployed in a VPC, enable the `native_gsuite`
+  feature flag in the dbt Cloud admin backend.
+
+### Supply your OAuth Client ID and Client Secret
+
+1. Navigate to the **Enterprise &gt; Single Sign On** page under Account
+Settings.
+2. Click the **Edit** button and supply the following SSO details:
+    - **Log in with**: GSuite
+    - **Client ID**: Paste the Client ID generated in the steps above
+    - **Client Secret**: Paste the Client Secret generated in the steps above
+    - **Domain in GSuite**: Enter the domain name for your GSuite account (eg. `fishtownanalytics.com`).
+      Only users with an email address from this domain will be able to log into your dbt Cloud
+      account using GSuite auth.
+    - **Slug**: Enter your desired login slug. Users will be able to log into dbt
+      Cloud by navigating to `https://cloud.getdbt.com/enterprise-login/<login-slug>`. Login slugs must
+      be unique across all dbt Cloud accounts, so pick a slug that uniquely
+      identifies your company.
+    <Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/gsuite/gsuite-sso-cloud-config.png" title="GSuite SSO Configuration"/>
+3. Click **Save &amp; Authorize** to authorize your credentials. You should be
+   dropped into the GSuite oauth flow and prompted to log into dbt Cloud with
+   your work email address. If authentication is successful, you will be
+   redirected back to the dbt Cloud application.
+4. On the **Verify SSO Credentials** page, verify that a `groups` entry is
+   present, and that it reflects the groups you are a member of in GSuite. If
```

Comment:
Does it make sense to link to the Enterprise permissions/say a little more about what the groups map to?

Reply:
that would be good information to include, but I don't think we need to say too much about it in this doc. I think we probably just need a new docs page that addresses group mappings in general. Let's do that in a separate PR

Reply:
That sounds great!